### ISSUE TITLE
Improve Macro Readability by Listing 'T4-The Rest' Divination Cards by Name

### ISSUE THREAD

**Issue by dsnvwlmnt** [NeverSinkDev/NeverSink-Filter#116]
Listing div cards as T1, T2, T3, T5, T4-The Rest is a bit confusing, and hard to follow what changes from version to version.

There's really not *that* many cards that get scooped up in T4-The Rest, so why not just list them out like the other sections, and in logical order T1 T2 T3 T4 T5?

This has a side-benefit as well, that if you guys miss a card, right now it gets scooped up as T4. If you list them all, a missed one would get scooped up at the end of your filter in pink.

Thanks for your consideration.

**Comment by NeverSinkDev** [comment]
I've been long considering implementing this, but ultimately decided against this. 

There's several up and downsides on every side. 

Ultimately what makes me dislike that idea is that users will be simply **removing** basetypes from an individual list and them ending up getting caught by the "New items tier".

That being said, we recently introduced a URL for tierlists, it's still work in progress, but I hope this helps you out a bit:

http://filterblade.xyz/TierLists

**Comment by dsnvwlmnt** [comment]
Yeah that seems useful. There doesn't seem to be previous changelogs though. And it's not shown from which to which version the current changelog refers to... Digging around it looks like it's for changes going from "NeverSink's itemfilter - version 7.13" to "NeverSink's itemfilter - version 7.2.0", i.e. "FilterBlade 0.9.11j" to "FilterBlade 0.9.11k"?

**Comment by NeverSinkDev** [comment]
Yeah, we'll see if we can expand the format to include history and additional information

**Comment by NeverSinkDev** [comment]
Note: this will be implemented for 3.10.

The new system has explicit tiers for all "remaining" tiers, extra sections for safety/outdated/new highlight, improved treatment of hidden items and other goodies.

The tiering link has been removed though.
